config/default.yml

```
@@ -49,6 +49,21 @@ Rails/ActionFilter:
   Include:
     - app/controllers/**/*.rb
 
+Rails/ActionOrder:
+  Description: 'Enforce consistent ordering of controller actions.'
+  Enabled: pending
+  VersionAdded: '2.13'
```

Can you specify `<<next>>` instead?

```suggestion
  VersionAdded: '<<next>>'
```